### ISSUE TITLE
Higher refresh rate support

### ISSUE THREAD

**Issue by SlowNicoFish** [brave/brave-browser#13462]
## Description 
Brave seems to be locked at 60hz, I would love for brave to support other refresh rates such as 90hz, 120hz and 144hz. For reference I have a Oneplus 8T.

## Actual result 
Locked refresh rate to 60hz

## Expected result
Higher refresh rate support.

## Issue reproduces how often 
Every time Brave is started.

## Version/Channel Information:

- Can you reproduce this issue with the current Play Store version? Yes
- Can you reproduce this issue with the current Play Store Beta version? Yes
- Can you reproduce this issue with the current Play Store Nightly version? Yes

## Device details
   - Install type (ARM, x86): Arm 64
   - Device type (Phone, Tablet, Phablet): Phone
   - Android version: 11
   
## Brave version
1.18.77

### Website problems only
- Does the issue resolve itself when disabling Brave Shields? No
- Does the issue resolve itself when disabling Brave Rewards? No
- Is the issue reproducible on the latest version of Chrome? 
No

**Comment by anthonypkeane** [comment]
Hi @SlowNicoFish Do you know if Chrome lets you do this? Thanks

**Comment by ShadowTux** [comment]
In chrome you can't  manually change the refresh rate but it automatically uses higher refresh speed. In my case (OnePlus 8 Pro) it will use 120hz. Unlike brave uses 60hz

**Comment by Draggie306** [comment]
Not having 120hz in Brave on my OP8T on Android 12 was enough to make me factory restore my phone to Android 11 just to force everything to run at 120hz using ADB commands, which drains my battery... please add this soon!

**Comment by SergeyZhukovsky** [comment]
any steps how do you measure refresh rate? I have Google Pixel6 and max it gives is 90Hz and I see 90Hz in Brave:
![Screenshot_20220825-144022](https://user-images.githubusercontent.com/12011303/186743658-ab401056-9efa-4b0c-8431-81cc009db46d.png)
It becomes 90 when you do page scrolling as higher refresh rate is needed for smooth scrolling.

**Comment by SergeyZhukovsky** [comment]
![Screenshot_20220825-221845_Brave - Nightly](https://user-images.githubusercontent.com/12011303/186750998-bfd0355b-d3dc-4821-b93f-9d2dbbe1cea6.jpg)
120Hz ^

**Comment by Draggie306** [comment]
![Screenshot_20220826-105536.jpg](https://user-images.githubusercontent.com/75524491/186880426-345600ed-970c-424b-9c2f-5625f267d24c.jpg) When 120hz is forced _everywhere on all apps_ with ADB

![Screenshot_20220826-105640.jpg](https://user-images.githubusercontent.com/75524491/186880346-7915607b-ef95-45c1-8a38-b7fe4437f3fe.jpg) When adaptive 120hz is selected in settings.

There is no change when scrolling. Maybe it's a OnePlus issue. The same is true on Beta and Nightly builds.

**Comment by SergeyZhukovsky** [comment]
@Draggie306 it could be that there is something special in OnePlus. I tried 90Hz on Pixel 6 and 120Hz @AlexeyBarabash tried on Samsung Galaxy S22 Ultra. Do you mind to check what does it show in Chrome?

**Comment by Draggie306** [comment]
@SergeyZhukovsky Chrome is fine, no issues running at 120hz

![Screenshot_20220826-143333.jpg](https://user-images.githubusercontent.com/75524491/186915703-122dbacc-1404-41fb-a3f7-20efcfeb9e6d.jpg)

MS Edge is the same as Chrome too.

**Comment by SergeyZhukovsky** [comment]
@Draggie306 would you mind to go to `chrome://gpu` on both Brave and Chrome and post screenshots? Could you use Brave Nightly for that?

**Comment by Draggie306** [comment]
@SergeyZhukovsky ![Screenshot_20220826-151611.jpg](https://user-images.githubusercontent.com/75524491/186924084-77d157aa-f445-46d6-a17b-d9094bcc0606.jpg) This is what Brave shows me (Page load failed because the developer UI module (dev_ui) is not installed)

![Screenshot_20220826-151539.jpg](https://user-images.githubusercontent.com/75524491/186924212-a150c823-7ad8-49d9-85ac-8e88963d93f7.jpg) Chrome shows this, I can upload a copy of the entire report if it's needed

**Comment by SergeyZhukovsky** [comment]
@Draggie306 thanks, I'm trying to figure out. Does edge show that page as well?

**Comment by Draggie306** [comment]
@SergeyZhukovsky Here's what Edge shows: ![Screenshot_20220826-153943.jpg](https://user-images.githubusercontent.com/75524491/186930490-40746292-7ac3-4199-8082-ead3719480f7.jpg)

all of edge's info: https://pastebin.com/vWcFGtub

chrome's info: https://pastebin.com/jkzHz3mA

**Comment by SergeyZhukovsky** [comment]
Thanks @Draggie306! Most likely the device is needed to figure out what is going on.

**Comment by SergeyZhukovsky** [comment]
Community reports as well https://community.brave.com/t/compatibility-with-120-hz-refresh-rate/119501

**Comment by SergeyZhukovsky** [comment]
I found that on some forums btw `A fair amount of [OnePlus](https://www.androidheadlines.com/2022/04/oneplus-nord-2t-leak) phones have displays that run at 120Hz. This is something that people like, but there are some caveats. The software has a list of certain apps that can use the full 120Hz.`

**Comment by SergeyZhukovsky** [comment]
We should knock to OnePlus in that case.

**Comment by PhantomPainX** [comment]
Yes, this is a problem from the OnePlus branch, tested in other phones and Brave runs smoothly in 120hz

**Comment by SergeyZhukovsky** [comment]
I pinged OnePlus and OnePlus_Support in that tweet, let's see will it help or not https://twitter.com/SergZh_/status/1563187733136560133

**Comment by xVemu** [comment]
Same issue on Realme phones.

**Comment by 0xGREG** [comment]
If you download AutoHz app and give it perms via adb you can force Brave to be 120hz, other than that I haven't found a solution.

**Comment by SergeyZhukovsky** [comment]
@timchilds no, we haven't. Upstream Chromium doesn't have such calls btw, but I can take a look if we can use it or not or does it make any differences at all.

**Comment by xVemu** [comment]
> @timchilds no, we haven't. Upstream Chromium doesn't have such calls btw, but I can take a look if we can use it or not or does it make any differences at all.

any news?

**Comment by SergeyZhukovsky** [comment]
@xVemu nothing done yet, but I doubt we can control that. Samsung and Pixel devices work as expected on 120Hz if they support it. Brands can lock it for particular apps, Brave isn't there if it shows lower resolution.

**Comment by sunnamed434** [comment]
same issue, if I'll try to open google chrome its moves a way better than Brave

**Comment by Sharanam** [comment]
I am using Realme P1 5G and facing the same issue. Chrome works with 120 FPS, Brave works at 60 FPS. Tried https://www.displayhz.com/ and found the same (varying between 59 and 61)
